# Message representing a LiDAR scanner configuration

# Number of beams in horizontal and vertical direction respectively
int32 horizontalBeams
int32 verticalBeams

# Seperation of beams in degree in horizontal and vertical direction respectively
float32 horizontalResolution
float32 verticalResolution